github/messages.go

```diff
@@ -190,7 +205,7 @@ func ValidatePayload(r *http.Request, secretToken []byte) (payload []byte, err e
 	// Only validate the signature if a secret token exists. This is intended for
 	// local development only and all webhooks should ideally set up a secret token.
 	if len(secretToken) > 0 {
-		sig := r.Header.Get(signatureHeader)
+		sig := payloadSignature(r)
```

Comment:
since payloadSignature is so short, it's only ever called from here, and as you note elsewhere we could probably drop support for SHA1 at some point anyway, maybe just check for the two headers inline here?

``` go
	sig = r.Header.Get(sha256SignatureHeader)
	if sig == "" {
		sig = r.Header.Get(sha1SignatureHeader)
	}
```

Reply:
Good point! Changed it to the inline version. Thanks for looking at this so quick :rocket:.